docs/integration/frameworks/set-up-with-nextjs.md

```diff
@@ -0,0 +1,95 @@
+---
+description: A step-by-step guide to integrate Keyshade into your Next.js app for managing secrets and environment variables — no .env files needed.
+---
+
+# How to Set Up Keyshade into a Next.js Project
+
+## 1. Create a Next.js Project
+
```

Comment:
Maybe you can mention that we are a drop-in replacement to your `.env` so that they know it is not something with extra configs